server/pages/index.jade

```diff
@@ -54,6 +54,8 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				link(rel='stylesheet', href=urls['style-debug.css'])
 			else
 				link(rel='stylesheet', href=urls['style.css'])
+		style(type='text/css' id='critical-path-css')
+			!= Array.from(css).join('')
```

Comment:
@tug I think this change will be needed for SSR so we might as well make it now. We can always revisit it if we spot other issues down the line.